packages/nextjs/app/configure/page.tsx

```diff
@@ -0,0 +1,18 @@
+import DownloadContracts from "./_components/DownloadContracts";
+import type { NextPage } from "next";
+import { getMetadata } from "~~/utils/scaffold-stark/getMetadata";
+
+export const metadata = getMetadata({
+  title: "Configure Contracts",
```

Comment:
It works fine.
![Captura desde 2024-08-29 09-56-35](https://github.com/user-attachments/assets/d64e9a65-2e0d-4c0d-9c40-3a303ad8de68)

But the title and description don't load, I see it like this.

![Captura desde 2024-08-29 09-41-20](https://github.com/user-attachments/assets/d421f0b7-3a74-4dbe-96a5-3c5e2227e20f)

Reply:
<img width="606" alt="image" src="https://github.com/user-attachments/assets/bfa0bded-8840-4ee9-8559-6bf210e2acba">

Sir, these codes are for `i18n` and are not directly displayed on the page. They will serve for crawler robots

Reply:
Ok perfect, you should adjust a title or a superior design, or at least separate the upper border.

Reply:
OK sir let me adjust the container padding and border margin.

Reply:
<img width="1109" alt="image" src="https://github.com/user-attachments/assets/ccbd2a53-1867-4b4e-97af-aa1fa7867701">

make the title bolder and adjust the padding and border margin.

Reply:
Here I propose for now to keep something similar DebugContract.tsx [L27](https://github.com/Quantum3-Labs/scaffold-stark-2/blob/main/packages/nextjs/app/debug/_components/DebugContracts.tsx#L27), it would follow the line better.

Now
![imagen](https://github.com/user-attachments/assets/a44e40a9-108e-4046-849a-a626156e5d98)

With changes
![imagen](https://github.com/user-attachments/assets/c4e96d71-c231-4287-9e1a-1585a197bcb9)



![imagen](https://github.com/user-attachments/assets/2e3afe12-9fac-4bc6-b98e-05043e1e521d)